scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb

```diff
@@ -4,9 +4,57 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Personal Financial Assistant\n",
+    "# Personal Financial Assistant"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
```

Comment:
Needs objective + link to documentation

Reply:
# Personal Financial Assistant
## Objective

This notebook demonstrates the following:

- Function Calling with Yfinance to get latest stock prices. Summarization of user provided article. Extract country info from article, extract country, capital and other aspects, and call an API to get more information about each country.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant)
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 10-15 minutes running this sample. 

## About this example
The objective of the provided Python file is to create an Azure OpenAI Assistant named "Portfolio Management Assistant" using the Azure OpenAI API. The assistant is designed to act as a personal financial assistant, providing information and insights related to a user's investment portfolio. The script initiates a conversation with the assistant, guiding it through various financial queries and scenarios to showcase its capabilities.

### Data
This sample uses files from the folder [`data/finance/`](../data/finace/) in this repo. You can clone this repo or copy this folder to make sure you have access to these files when running the sample.

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/ca2359e0ec502e2143942885df0ad0c2ebcbfa47